examples/algorithms/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(example_programs
 
 foreach(example_program ${example_programs})
 
-  set(${example_program}_FLAGS DEPENDENCIES iostreams_component)
```

Comment:
on master this line looks like the following:

set(${example_program}_FLAGS DEPENDENCIES iostreams_component boost_regex)

Reply:
@ct-clmsn I'm not sure I understand what you mean by that. Care to elaborate?

Reply:
Had a couple compile errors this week after doing several synchs with master. Isolated the issue down to that line in CMakeLists.txt. Made the modification and found boost_regex isn't in the master CMakeLists.txt...after submitting the PR. I'll close this PR.